### DISCUSSION TITLE
Contribution Guide

### DISCUSSION THREAD

**Discussion by yasindce1998** [aquasecurity/tracee#3793] [Development]
I am trying to find contribution guide for trivy using this link: https://github.com/aquasecurity/tracee/blob/main/contributing/overview

It does exist and also I found this contribution link for over all aquasecurity: https://github.com/aquasecurity/.github/blob/master/CONTRIBUTING.md

can we follow the overall link or trivy has any specific contribution method?

**Comment by itaysk** [comment]
Hi, looks like you found the link for Tracee https://github.com/aquasecurity/tracee/pull/3794/
In short, feel free to make any contribution but please before you do open an Issue/Discussion to explain the motivation and design before you send a PR. Thanks!
